src/core/DataFile.cpp

```diff
@@ -1385,6 +1387,44 @@ void DataFile::upgrade_noHiddenClipNames()
 	}
 }
 
+void DataFile::upgrade_noHiddenAutomationTracks()
+{
+	// convert global automation tracks to non-hidden
+	QDomElement song = firstChildElement("lmms-project")
+		.firstChildElement("song");
+	QDomElement tc = song.firstChildElement("trackcontainer");
+	QDomElement gaTrack = song.firstChildElement("track");
+	if (!gaTrack.isNull()
+		&& gaTrack.attribute("type").toInt() == Track::HiddenAutomationTrack)
+	{
+		// global automationpatterns
+		QDomNodeList aps = gaTrack.elementsByTagName("automationpattern");
```

Comment:
This should be changed to `"automationclip"` now.

Reply:
Done!